components/popover/index.js

```diff
@@ -331,4 +335,7 @@ Popover.contextTypes = {
 
 Popover.Slot = () => <Slot bubblesVirtually name={ SLOT_NAME } />;
 
+// Make this available for use by other modal UI that is not based on Popover
+Popover.MobileScrollLock = MobileScrollLock;
```

Comment:
If we want to make it reusable for other use cases it should be exposed as its own component to avoid confusion. I was sure it is specific to the Popover component.

Reply:
I think you're right and will make the change. The original thinking was that it was ideally only for popover but could be used for Popover-like components.